src/OpenTelemetry.AutoInstrumentation.StartupHook/OpenTelemetry.AutoInstrumentation.StartupHook.csproj

```diff
@@ -4,4 +4,10 @@
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\OpenTelemetry.AutoInstrumentation\Constants.cs" Link="Constants.cs">
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```

Comment:
Is this really needed for a `Compile` item?

Reply:
Fixed.